hazelcast/src/main/java/com/hazelcast/spi/merge/MergeDataHolder.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.spi.merge;
+
+import com.hazelcast.nio.serialization.DataSerializable;
+import com.hazelcast.spi.serialization.SerializationService;
+
+/**
+ * Represents a read-only view of data structure entries for the merging process after a split-brain.
```

Comment:
typo in `data structure entries`?  this is only for `values` now

Reply:
Good catch, I've also fixed `KeyMergeDataHolder` to
> Represents a read-only view of a data structure key/value-pair for the merging process after a split-brain.